source/src/molecular-unfolding/lambda/TaskParametersLambda/app.py

```diff
@@ -64,6 +47,8 @@
 max_mem, min_mem = 30,  1
```

Comment:
remove it if it's useless

Reply:
we should keep this value, as the max instance launched is EC2.C5.XLARGE4(16 vcpus, 32G mem), if set mem > 30, there is no instance to run the task.